pom.xml

```diff
@@ -44,7 +44,7 @@
         <!--Dependencies -->
         <gherkin.version>5.1.0</gherkin.version>
         <cucumber-expressions.version>8.0.0</cucumber-expressions.version>
-        <datatable.version>2.0.0</datatable.version>
+        <datatable.version>3.0.0-SNAPSHOT</datatable.version>
```

Comment:
I see the build is failing because there isn't a 3.0.0-SNAPSHOT published.

You should be able to do this easily with `make deploy` from `datatable/java`

Reply:
I just did a `make deploy` from the `reimplement-type-factory` branch, which I'm guessing is the only branch targetting `3.0.0-SNAPSHOT`.

It's a little tedious not to have snapshots continuously deployed. OTOH it's a bit more secure, and it's relatively easy to do a manual snapshot release when needed.

Then I triggered a new build, which [now passes](https://travis-ci.org/cucumber/cucumber-jvm/builds/572511667) 💚 🥒

Reply:
Cheers!